syntax = "proto2";
import "game_commands.proto";
message Command_ReverseTurn {
    extend GameCommand {
        optional Command_ReverseTurn ext = 1034;
    }
}
